docs/source/contributing/developer_guide_implementing_distribution.md

```diff
@@ -350,8 +364,56 @@ def test_blah_logcdf(self):
 
 ```
 
+## 5. Adding tests for the `get_moment` method
+
+Tests for the `get_moment` method are contained in `pymc/tests/test_distributions_moments.py`, and check if:
+1. Moments return `expected` values 
+1. Moments have the expected size and shape
+
+```python
+
+import pytest
+from pymc.distributions import Blah
+from pymc.initial_point import make_initial_point_fn
+
+def assert_moment_is_expected(model, expected):
```

Comment:
We shouldn't copy the body of `assert_moment_is_expected`, just mention it exists, what it does and and how it should be used (which you show below)

Reply:
Thanks for the review! That makes sense if eventually the function `assert_moment_is_expected` changes in the test file we don't have to track the changes and update the documentation. I will work on these changes a commit them soon.